### PR TITLE
Parties page styling

### PR DESCRIPTION
**Pull request by MikeNGarrett**
Quite a few changes to make picking a party better than ever. Let me know if I missed any states for styling.

### CODE REVIEW

**Comment by superemily** [comment]
It looks like this:

screen shot 2015-05-13 at 12 13 49 am

File: rdiodj/static/js/partylist.js [hunk 1]
```diff
@@ -87,7 +90,9 @@ app.RoomView = Backbone.View.extend({
           }
         });
       } else {
-        data.nowPlaying = "silence";
+        data.nowPlaying = "sadness";
```
Comment (superemily) [inline rdiodj/static/js/partylist.js]: I find "sadness" here a little bit confusing. What's wrong with "silence"?

Comment (MikeNGarrett) [inline rdiodj/static/js/partylist.js]: Feel free to change it back. I thought it was sad that a room existed without playing music.

**Comment by MikeNGarrett** [comment]
For the blue, it's your call. I tried to match something closer to Rdio. I don't know if that's the right direction, but I think the experience outside of the party room needs a bit of color. 

Also, the dark boxes behind the Parties Already Started are filled with the currently playing album art when people are playing something.

**Comment by superemily** [comment]
Well, if Marek is ok with the death of his banana, I say let's go for it.

**Pull request merged**